api/samples/CMakeLists.txt

```diff
@@ -162,8 +161,7 @@ function (add_sample_client name source_file_list extension_list)
 
   # Provide a hint for how to use the client
   if (NOT DynamoRIO_INTERNAL OR NOT "${CMAKE_GENERATOR}" MATCHES "Ninja")
-    # XXX i#1557: replace w/ generator expression to satisfy CMP0026
-    get_target_property(path ${name} LOCATION${location_suffix})
+    DynamoRIO_get_full_name(path_test ${name})
```

Comment:
I think you already put a comment here: s/path_test/path/.  Please verify visually that the resulting printout is correct as there is no automated check.

Reply:
Done